### PR TITLE
ERA-780 Fix Dockers in Refapps

### PR DESCRIPTION
**Pull request by tamaskalcza-da**

### CODE REVIEW

File: Dockerfile-sandbox [hunk 1]
```diff
@@ -4,16 +4,23 @@
...
 
-CMD ~/.daml/bin/daml sandbox -- --address 0.0.0.0 --port 6865 --scenario Test.Test:onboarding inventory-management.dar
+ENTRYPOINT daml start \
+  --sandbox-option="--address=0.0.0.0" \
+  --sandbox-port 6865 \
+# Cannot explicitly specify, because of: https://github.com/digital-asset/daml/issues/5777
+# Relying on default port behaviour as of now.
+#  --navigator-option="--port=7500" \
+  --open-browser=no \
+  --json-api-port=none
```
Comment (richardkapolnai-da) [inline Dockerfile-sandbox:26]: nice

File: Dockerfile-sandbox [hunk 2]
```diff
@@ -4,16 +4,23 @@
...
 
-USER root
-RUN chown daml inventory-management.dar
-USER daml
+FROM source
 
 EXPOSE 6865
+EXPOSE 7500
 
-CMD ~/.daml/bin/daml sandbox -- --address 0.0.0.0 --port 6865 --scenario Test.Test:onboarding inventory-management.dar
```
Comment (richardkapolnai-da) [inline Dockerfile-sandbox:19]: why don't we need this scenario?

Comment (tamaskalcza-da) [reply Dockerfile-sandbox:19]: It's move to script and it's in `daml.yaml`.

Comment (gaborh-da) [reply Dockerfile-sandbox:19]: Yes that's a valid question. And don't we have a script instead? (I remember writing something with this name)

**Pull request merged**